crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -543,8 +543,8 @@ impl<'a> SimpleTokenizer<'a> {
                 kind
             }
 
-            ' ' | '\t' => {
-                self.cursor.eat_while(|c| matches!(c, ' ' | '\t'));
+            ' ' | '\t' | '\x0C' => {
```

Comment:
Please add a comment explaining that we ignore the true semantics of formfeed here ...